acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -3,6 +3,13 @@
 
 extend Puppet::Acceptance::InstallUtils
 
+# Force clock skew, note agent can't be more than 5 minutes
```

Comment:
I could go either way. It's handy so I don't have to remember how to do that, but if it's distracting I can remove.